app/controllers/vans_controller.rb

```diff
@@ -0,0 +1,34 @@
+class VansController < ApplicationController
+  before_action :set_task, only: [:show, :update]
+
+  def index
+    @vans = Van.all
+  end
+
+  def show; end
+
+  def new
+    @van = Van.new
+  end
+
+  def create
+    van = Van.new(van_params)
+    van.save
```

Comment:
I would add in an if statement with: if @van.save -> redirect_to van_path(@van)
else -> render :new, status: :unprocessable_entity